script/fork-helpers/NodeOperators.s.sol

```diff
@@ -0,0 +1,196 @@
+            0
+        );
+        _setBalance(penaltyReporter, "1000000000000000000");
+        vm.startBroadcast(penaltyReporter);
+        _;
+        vm.stopBroadcast();
+    }
+
+    modifier broadcastPenaltySettler() {
+        _setUp();
+        address penaltySettler = csm.getRoleMember(
+            csm.SETTLE_EL_REWARDS_STEALING_PENALTY_ROLE(),
+            0
+        );
+        _setBalance(penaltySettler, "1000000000000000000");
+        vm.startBroadcast(penaltySettler);
+        _;
+        vm.stopBroadcast();
+    }
+
+    modifier broadcastVerifier() {
+        _setUp();
+        _setBalance(address(verifier), "1000000000000000000");
+        vm.startBroadcast(address(verifier));
+        _;
+        vm.stopBroadcast();
+    }
+
+    modifier broadcastStakingRouter() {
+        _setUp();
+        _setBalance(address(stakingRouter), "1000000000000000000");
+        vm.startBroadcast(address(stakingRouter));
+        _;
+        vm.stopBroadcast();
+    }
+
+    function deposit(uint256 depositCount) external broadcastStakingRouter {
+        (, , uint256 depositableValidatorsCount) = csm
+            .getStakingModuleSummary();
+        if (depositCount > depositableValidatorsCount) {
+            depositCount = depositableValidatorsCount;
+        }
+        csm.obtainDepositData(depositCount, "");
+    }
+
+    function unvet(
+        uint256 noId,
+        uint256 vettedKeysCount
+    ) external broadcastStakingRouter {
+        csm.decreaseVettedSigningKeysCount(
+            bytes.concat(bytes8(uint64(noId))),
+            bytes.concat(bytes16(uint128(vettedKeysCount)))
+        );
+    }
+
+    function exit(
+        uint256 noId,
+        uint256 exitedKeysCount
+    ) external broadcastStakingRouter {
+        csm.updateExitedValidatorsCount(
+            bytes.concat(bytes8(uint64(noId))),
+            bytes.concat(bytes16(uint128(exitedKeysCount)))
+        );
+    }
+
+    function stuck(
+        uint256 noId,
+        uint256 stuckKeysCount
+    ) external broadcastStakingRouter {
+        csm.updateExitedValidatorsCount(
+            bytes.concat(bytes8(uint64(noId))),
+            bytes.concat(bytes16(uint128(stuckKeysCount)))
+        );
+    }
+
+    function withdraw(
+        uint256 noId,
+        uint256 keyIndex,
+        uint256 amount
+    ) external broadcastVerifier {
+        csm.submitWithdrawal(noId, keyIndex, amount);
+    }
+
+    function slash(uint256 noId, uint256 keyIndex) external broadcastVerifier {
+        csm.submitInitialSlashing(noId, keyIndex);
+    }
+
+    function targetLimit(
+        uint256 noId,
+        uint256 targetLimitMode,
+        uint256 limit
+    ) external broadcastStakingRouter {
+        csm.updateTargetValidatorsLimits(noId, targetLimitMode, limit);
+    }
+
+    function reportStealing(
+        uint256 noId,
+        uint256 amount
+    ) external broadcastPenaltyReporter {
+        csm.reportELRewardsStealingPenalty(noId, "", amount);
```

Comment:
hash might be useful for notification testing, i guess. let's set some mocked value